osf/management/commands/backfill_domain_references.py

```diff
@@ -65,8 +66,11 @@ def backfill_domain_references(model_name, dry_run=False, batch_size=None, ignor
         logger.info(f'{item}, queued')
         spam_content = item._get_spam_content(include_tags=False)
         if not dry_run:
+            item_id = item._id
+            if VersionedGuidMixin.GUID_VERSION_DELIMITER in item_id:
+                item_id = item_id.split(VersionedGuidMixin.GUID_VERSION_DELIMITER)[0]
             spam_tasks.check_resource_for_domains_async.apply_async(
-                kwargs={'guid': item._id, 'content': spam_content}
+                kwargs={'guid': item_id, 'content': spam_content}
```

Comment:
This is a similar case. Your fix provides the base guid to `check_resource_for_domains_async` ... this fixes the tests but break the actual code.

The actual fix should be update `check_resource_for_domains_async` to handle both non-versioned and versioned guid.